docs/conf.py

```diff
@@ -252,7 +252,40 @@ def copy_doc(src, dest, title=""):
 
 
 def setup(app):
-    pass
+    # Only connect the hook for the "latest" version on Read the Docs
+    # to avoid generating llms.txt for every version build
+    # TODO: change to stable (before releasing 1.8.0)
+    if environ.get("READTHEDOCS_VERSION") == "latest":
+        app.connect("build-finished", create_llms_txt)
```

Comment:
why is it not stable now?

Reply:
had it coming for the verbosity
what I'm trying to understand is what's the plan and why
if latest is just for testing then should we always build for testing?
if not use both latest and stable?
when is this triggered? on push to stable or on build on readthedocs?

Reply:
thanks 🙏 we might want to test against 1.8.x before releasing instead of latest to minimize the risk

Reply:
@alonmr okay, @jillnogold fyi